src/wp-admin/includes/dashboard.php

```diff
@@ -302,22 +302,25 @@ function wp_dashboard_right_now() {
 	<div class="main">
 	<ul>
 	<?php
-	// Posts and Pages.
-	foreach ( array( 'post', 'page' ) as $post_type ) {
-		$num_posts = wp_count_posts( $post_type );
-
-		if ( $num_posts && $num_posts->publish ) {
-			if ( 'post' === $post_type ) {
-				/* translators: %s: Number of posts. */
-				$text = _n( '%s Post', '%s Posts', $num_posts->publish );
-			} else {
-				/* translators: %s: Number of pages. */
-				$text = _n( '%s Page', '%s Pages', $num_posts->publish );
-			}
+	// At a Glance Post Types.
+	foreach ( get_post_types( array( 'at_a_glance' => true ) ) as $post_type ) {
+		$num_posts          = wp_count_posts( $post_type );
+		$num_post_published = intval( $num_posts->publish );
 
-			$text             = sprintf( $text, number_format_i18n( $num_posts->publish ) );
+		if ( $num_posts && $num_post_published ) {
 			$post_type_object = get_post_type_object( $post_type );
 
+			if ( ! $post_type_object ) {
+				continue;
+			}
+			if ( 1 === $num_post_published ) {
+				$post_label = $post_type_object->labels->singular_name;
+			} else {
+				$post_label = $post_type_object->labels->name;
+			}
+			/* translators: %d: Number of posts, %s post label. */
+			$text = sprintf( '%d %s', number_format_i18n( $num_post_published ), $post_label );
```

Comment:
The translator comment isn't relevant because the string `%d %s` isn't translated.

So it can be just:


```suggestion
			$text = number_format_i18n( $num_post_published ) . ' ' . $post_label;
```